variables.tf

```diff
@@ -222,6 +222,12 @@ variable "iam_path" {
   default     = "/"
 }
 
+variable "cluster_endpoint_private_access_cidrs" {
+  description = "List of CIDR blocks which can access the Amazon EKS private API server endpoint, when public access is disabled"
+  type        = list(string)
+  default     = ["127.0.0.1/32"]
```

Comment:
Wouldn't `0.0.0.0/32` make more sense here? Because if `cluster_endpoint_private_access` is true and `cluster_endpoint_public_access` is false, there is ONLY access within the VPC anyway. With this default config, nothing will be able to connect to the endpoint.

Reply:
good catch,   I set it to 127.0.0.1 to restict access by default but when you set only private, it gives the eks endpoints a private only address to setting it to 0.0.0.0 still makes it inaccessible to the world.    updated.